README.md

```diff
@@ -36,22 +36,31 @@ jobs:
         with:
           token: ${{ github.token }}
           branch: main
+          major-list: break
+          minor-list: feat, feature
+          patch-list: fix, bugfix, perf, refactor, test, tests
```

Comment:
Don't include optional fields in the example, as most users will blindly copy-paste and think it's required. Makes it more difficult to change / add to the defaults in the future if needed.